src/frontend/components/UI/QuestDetails/index.tsx

```diff
@@ -0,0 +1,140 @@
+import React, { useCallback } from 'react'
+import { QuestDetailsWrapper } from '@hyperplay/quests-ui'
+import { Reward, Quest } from '@hyperplay/utils'
+import useAuthSession from 'frontend/hooks/useAuthSession'
+import useGetUserPlayStreak from 'frontend/hooks/useGetUserPlayStreak'
+import authState from 'frontend/state/authState'
+import { useFlags } from 'launchdarkly-react-client-sdk'
+import { useTranslation } from 'react-i18next'
+import { useAccount } from 'wagmi'
+import { useSyncPlayStreakWithExternalSource } from 'frontend/hooks/useSyncPlayStreakWithExternalSource'
+
+/**
+ * Don't delete this comment block since it's used for translation parsing for keys that are on the quests-ui library.
+ * As a heads up, everytime you add a new key on any library, you need to add it as a block comment anywhere in the code as well.
```

Comment:
That's a great idea, actually. I tried adding that pattern to the config, but i18n-next-parser is not detecting the usage of the `useTranslation` because the content of the quests-ui is minified. I guess it is because the package expects a text match for `t('**')`. 

![image](https://github.com/user-attachments/assets/031b1dd7-69a1-48dd-8bf0-15dc59c31df7)


From what I see in the docs, the text match is not configurable, and I don't think we could use it anyways because the minified code is not predictable. Looking at the code, the `t('code')` is replaced with different letters in each usage.

![image](https://github.com/user-attachments/assets/25246d44-53c5-4530-983b-c7833c8dd724)
![image](https://github.com/user-attachments/assets/93202ff3-f5cf-49f3-a5cc-239c26afb177)

Reply:
Further investigation is out of the scope for this PR but def worth checking.